### PR TITLE
merging two pages, had to fix alignment

### PR DESCRIPTION
**Pull request by YosefAmin**

### CODE REVIEW

File: src/screens/tournaments/CreateTournamentScreen.js [hunk 1]
```diff
@@ -90,6 +90,7 @@ const CreateTournamentScreen = () => {
 
     // Returns date
     const getDate = (date) => {
+        console.log(date);
```
Comment (halvtomat) [inline src/screens/tournaments/CreateTournamentScreen.js:93]: Remove console log before pushing!!!!

**Review by halvtomat — CHANGES_REQUESTED** [review]
See comment

File: src/screens/tournaments/CreateTournamentScreen.js (hunk 1 above)
Comment (YosefAmin) [reply src/screens/tournaments/CreateTournamentScreen.js]: fixed m8

**Review by halvtomat — APPROVED** [review]
After the recent changes I have decided that this pull request is of extraordinary quality. Approved

**Pull request merged**